index.html

```diff
@@ -99,6 +109,93 @@ <h2>Multi-Post <br> Stories </h2>
         </ul>
         <button>See Project</button></div>
     </section>
+    <!Desktop version>
+    <section class="desktop-work-sec">
+        <div class="desk-first-div">
+            <div class="desk-div-img"><img src="images/Snapshoot Portfolio (4).png"></div>
+            <div class="div-info">
+                <h2>Tonic</h2>
+                <ul class="ul-li">
+                    <li>CANOPY</li>
```

Comment:
- [ ] Please 🙏  make the cooler of the company in your card `black` instead of `grey` as in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0) it is also `black` ⚫ 

**_Figma Design_**

<img width="56" alt="Screenshot 2023-02-23 at 9 37 25 PM" src="https://user-images.githubusercontent.com/44798044/220980710-df2f0292-450d-4a71-93a5-d6cf7c3b4c67.png">

**_Your Design_**

<img width="106" alt="Screenshot 2023-02-23 at 9 37 14 PM" src="https://user-images.githubusercontent.com/44798044/220980744-31e5505c-cefd-49c7-819e-669650874b89.png">